pkg/plugins/addon/addon.go

```diff
@@ -66,7 +67,7 @@ func (c *AddOn) Description() string {
 	return description
 }
 
-func (c *AddOn) Score(ctx context.Context, placement *clusterapiv1beta1.Placement, clusters []*clusterapiv1.ManagedCluster) (map[string]int64, error) {
+func (c *AddOn) Score(ctx context.Context, placement *clusterapiv1beta1.Placement, clusters []*clusterapiv1.ManagedCluster) (map[string]int64, *time.Duration, error) {
```

Comment:
I think we should have a separate func to return requeue time. Having it in score looks weird. This func should be a common func for addon interface that both filter/score addon should implement.

Reply:
Just add a RequeueAfter() function to the interface.

Reply:
Unlike name and description, requeueAfter is not a property of a Plugin itself. Should it be part of Plugin interface?